schema/sql/api.sql

```diff
@@ -125,6 +126,23 @@ comment on column v1.ssh_public_key.uid is
 alter view v1."ssh_public_key" owner to api;
 grant select on table v1."ssh_public_key" to "api-anon";
 
+create view v1.gpg_public_key as
+  select
+      encode(fingerprint, 'hex') as fingerprint,
```

Comment:
should we also include the short key ID and long key ID?

Reply:
no:

- https://searchsecurity.techtarget.com/news/450302716/PGP-collision-attack-on-Linux-creator-highlights-flaws-with-short-IDs
- https://nullprogram.com/blog/2019/07/22/